noir_stdlib/src/ops/mod.nr

```diff
@@ -1,5 +1,5 @@
-mod arith;
-mod bit;
+pub mod arith;
+pub mod bit;
```

Comment:
```suggestion
mod arith;
mod bit;
```

We reexport anything of interest from these modules so we can keep these private.